supertokens_python/recipe/emailpassword/recipe.py

```diff
@@ -88,9 +90,10 @@ def __init__(self, recipe_id: str, app_info: AppInfo,
         if email_verification_recipe is not None:
```

Comment:
make ingredients changes for __init__ function here too similar to the email verification recipe